### DISCUSSION TITLE
修复下 components/message 的bug吧，不能显示效果啊

### DISCUSSION THREAD

**Discussion by zhiye** [Tencent/tdesign#485] [Q&A]
https://tdesign.tencent.com/miniprogram/components/message
该页面的预览效果 多个浏览器试了下，都不能显示了
赶紧修复下bug把

**Comment by anlyyao** [comment]
> https://tdesign.tencent.com/miniprogram/components/message 该页面的预览效果 多个浏览器试了下，都不能显示了 赶紧修复下bug把

哪个版本呢？ 1.3.1修复了一波，先试试1.3.1版本

**Comment by zhiye** [comment]
打开页面 https://tdesign.tencent.com/miniprogram/components/message&nbsp;右下角&nbsp;预览效果是空白啊









------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "Tencent/tdesign"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2024年5月10日(星期五) 下午2:39
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [Tencent/tdesign] 修复下 components/message 的bug吧，不能显示效果啊 (Discussion #485)





  
https://tdesign.tencent.com/miniprogram/components/message 该页面的预览效果 多个浏览器试了下，都不能显示了 赶紧修复下bug把
  
哪个版本呢？ 1.3.1修复了一波，先试试1.3.1版本

**Reply by anlyyao** [reply]
官网预览窗口的打包有点问题，在处理中。建议先用真机体验，不影响组件本身功能

**Comment by zhiye** [comment]
https://tdesign.tencent.com/miniprogram/components/back-top?tab=demo
页面上

BackTop 返回顶部 放入小程序 没有效果啊，怎么回事啊






把上图中 Component画红圈的改为page
还是没有效果啊？？




求助&nbsp; SOS



------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "Tencent/tdesign"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2024年5月10日(星期五) 下午2:39
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [Tencent/tdesign] 修复下 components/message 的bug吧，不能显示效果啊 (Discussion #485)





  
https://tdesign.tencent.com/miniprogram/components/message 该页面的预览效果 多个浏览器试了下，都不能显示了 赶紧修复下bug把
  
哪个版本呢？ 1.3.1修复了一波，先试试1.3.1版本

**Reply by anlyyao** [reply]
<img width="790" alt="截屏2024-05-13 16 24 46" src="https://github.com/Tencent/tdesign/assets/51158141/795dfb7b-ddee-4284-a4e8-7da4df0cead9">

- Page({})构造器里对应加一下这两个事件

**Comment by anlyyao** [comment]
小程序端问题建议直接去[仓库](https://github.com/Tencent/tdesign-miniprogram)下提问，响应速度比主仓高
